ontology/yaml/resources/HVAC/entity_types/FCU.yaml

```diff
@@ -2149,6 +2149,16 @@ FCU_DFSS_DFRMM_CSP_CHWDC:
   - CSP
   - CHWDC
 
+FCU_DFSS_ZTC_CHWISOVM:
```

Comment:
Not acceptable - the chilled water valve should be modeled via an abstract type (e.g., CHWZTC)

Reply:
The chilled water is a status point with the state open /close for this project but the command in the abstract type CHWZTC is a percentage command